.github/actions/test_tests-ldap.sh

```diff
@@ -1,14 +1,5 @@
 #!/bin/bash
 set -e -u -x -o pipefail
 
-vendor/bin/atoum \
-  -p 'php -d memory_limit=512M' \
-  --debug \
-  --force-terminal \
-  --use-dot-report \
-  --bootstrap-file tests/bootstrap.php \
-  --fail-if-void-methods \
-  --fail-if-skipped-methods \
-  --no-code-coverage \
-  --max-children-number 1 \
-  -d tests/LDAP
+
+vendor/bin/phpunit --no-coverage phpunit/LDAP/
```

Comment:
1. By default there is no coverage
2. Adding ` $@` permits to pass additional args when using this script locally.

```suggestion
vendor/bin/phpunit phpunit/LDAP $@
```